### PR TITLE
volume scheduling: move metrics to a separate package to avoid import cycle

### PR DESCRIPTION
**Pull request by cofyc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

I want to convert `pkg/controller/volume/scheduling` to a scheduler plugin using the new framework (https://github.com/kubernetes/kubernetes/issues/56180). However, `pkg/scheduler/framework/v1alpha1` imports `pkg/scheduler/util` which imports `pkg/scheduler/metrics`, and then imports `pkg/controller/volume/scheduling`. There will be an import cycle. 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by wgliang — APPROVED** [review]
/lgtm

File: pkg/controller/volume/scheduling/metrics/metrics.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```
Comment (Sindica) [inline pkg/controller/volume/scheduling/metrics/metrics.go:14]: Pay attention to copyright header of this file. Should not be Arktos only.

**Pull request merged**